app.py

```diff
@@ -31,6 +31,39 @@
 
     return "5cb85c"
 
+def get_style_fields(style):
+    style_templates = {
+        "default": {
+            "title": "",
+            "title_width": 0,
+            "title_color": "428bca",
+            "scale": 100,
+            "progress_width": 60,
+            # "progress_color": "5cb85c",
+            "progress_background": "555",
+            "progress_number_color": "fff",
+            "prefix": "",
+            "suffix": "%",
+            "show_text": True,
+            "show_shadow": True,
+            "border_radius": 4,
+        },
```

Comment:
the value of progress color is given by a function that makes use of progress value for its color. Uncommenting this, the value will ignore that function's return val, as if its given by a user when it is not. I only kept it here so that all the fields are viewable from the 'default' style_template, where the other styles might only contain what they need